src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -600,6 +600,7 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
                     ->canBeEnabled()
                     ->children()
                         ->scalarNode('storage_id')->defaultValue('session.storage.native')->end()
+                        ->scalarNode('storage_factory_id')->defaultNull()->end()
```

Comment:
i think `storage_id` and `storage_factory_id` should be mutually exclusive then

Reply:
Yes. this logic is handled in FrameworkExtension.